CONTRIBUTING.md

```diff
@@ -21,71 +21,69 @@ See [GitHub's official documentation](https://help.github.com/articles/using-pul
 
 ## Getting Started
 
-This repo contains source code, package build specifications for building the source into VI packages, and examples showing how to write and run MeasurementLink services.
+This repo contains source code, package build specifications for building the source into VI packages, and examples showing how to write and run measurement plug-ins.
 
 The source code can be found under the `Source` directory. The package definition files (.vipb) can be found under the `Build Specs` directory. The example code can be found under the `Example Measurements` directory.
 
 ## Building the LabVIEW Packages
 
 The source code is built into five LabVIEW packages
 
-* `ni_measurementlink_service`
-* `ni_measurementlink_generator`
+* `ni_measurement_plugin_sdk`
```

Comment:
Note: This LV package is named `*_plugin_*` whereas the others are `*_plug_in_*`